core/trino-main/src/main/java/io/trino/execution/CreateViewTask.java

```diff
@@ -76,6 +80,22 @@ public String getName()
 
         accessControl.checkCanCreateView(session.toSecurityContext(), name);
 
+        boolean isView = metadata.isView(session, name);
+        if (isView && !statement.isReplace()) {
+            throw semanticException(ALREADY_EXISTS, statement, "View already exists: '%s'", name);
+        }
+        else if (metadata.isMaterializedView(session, name)) {
```

Comment:
Change the order:
* MV
* view
* table
This is the metadata check order that we use in other places. Let's be consistent here

Reply:
@sopel39 I had changed the order,can you reviewer again